theme/themes/pastanaga/elements/segment.overrides

```diff
@@ -108,3 +108,7 @@
   border: 3px dashed #ccc;
   box-shadow: none;
 }
+
+.ui.clearing.segment.actions {
```

Comment:
@nileshgulia1 we just can't assume that all the segments clearing actions will be all uppercase... please use the scoped `.users-control-panel` for it.

Reply:
@sneridagh Oops sorry, I missed that! :)